src/styles/carousel.scss

```diff
@@ -18,10 +18,8 @@ $fd-carousel-dot-active-size: 0.5rem !default;
   max-width: 100%;
 
   @include fd-hover() {
-    .#{$block}__content {
-      .#{$block}__button {
-        display: block;
-      }
+    .#{$block}__content .#{$block}__button {
+      display: block;
```

Comment:
what do you achieve with this change? We usually nest the children elements like this because it's easier (visually) to see the number of nesting levels. BEM suggests we don't use more than 3 levels, that's why nesting them like it was done before allows you to easily determine these levels.

Reply:
oh, accidentally added this. reverted.